### DISCUSSION TITLE
Webpack 5 doesnt generate the chunks of imports

### DISCUSSION THREAD

**Discussion by pawreddy** [webpack/webpack#18175] [Q&A]
# Bug report

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Other relevant information:**
webpack version:  
Node.js version:
Operating System:
Additional tools:

**Comment by pawreddy** [comment]
Hi Team, 

I have webpack 5 installed for a react project, I observed that the webpack version 5 is not generating the chunk.js files under js folder for the imports of all the files from the entry point, is this a bug or am i missing something ?

**Comment by alexander-akait** [comment]
Please create a reproducible test repo, thank you

**Reply by pawreddy** [reply]
thanks @alexander-akait for reaching out, whats the new way to generate chunks for all the imports starting from the endpoint ?

**Reply by alexander-akait** [reply]
Sorry, we don't have an option for this, because our main goal is bundling and optimization, so we don't keep structure as is, but we have an issue for this

**Reply by pawreddy** [reply]
in webpack v4, it use to generate chunks, when we specify under optimisation with chunks: "all", is there now way we could do the same in Webpack 5 ?

**Reply by alexander-akait** [reply]
You can use the same option

**Reply by pawreddy** [reply]
I did, but unfortunately I could not see any other chunks that was generated when i used the same option, BTW I have used npm run eject to install the webpack, and have added the optimisation options for chunking and code split, but didnt work as expected as how it did in webpack V4

**Reply by alexander-akait** [reply]
Sorry, without reproducible test repo I can't help, I don't know your configuration
